### DISCUSSION TITLE
Should i686-unknown-linux-musl be listed as a supported target?

### DISCUSSION THREAD

**Discussion by clintfred** [cross-rs/cross#1282] [Q&A] [answered]
I noticed that `i686-unknown-linux-musl` is not listed as a supported target, but does appear to work on my project for `cross test`.

I was previously testing against `i586-unknown-linux-musl` (which is a supported target) and decided to try the i686 equivalent since i686 seems more popular than i586.

Is `i686-unknown-linux-musl` not supported? Is the table wrong? Is there a better target to use for just testing "general" 32-bit support for my application?

**Comment by Emilgardis** [comment, marked as answer]
Yes! It should be listed and is supported, thats a mistake on our part! Feel free to open a PR to fix it, or I'll get to it myself eventually :)
